src/test/java/org/isf/dicom/Tests.java

```diff
@@ -262,25 +262,25 @@ public void testSourceFilesPreloadDicomTest16() throws Exception {
 	}
 
 	@Test
-	public void testSourceFilesPreloadBadDicomFormat() throws Exception {
+	void testSourceFilesPreloadBadDicomFormat() throws Exception {
 		File file = getFile("BadDicomFile.dcm");
 		assertThatThrownBy(() -> new DicomInputStream(file))
 				.isInstanceOf(DicomStreamException.class);
 	}
 
 	@Test
-	public void testSourceFilesCountFiles() throws Exception {
+	void testSourceFilesCountFiles() throws Exception {
 		File file = getFile("dicomdir");
 		int count = SourceFiles.countFiles(file, 1);
 		assertThat(count).isPositive();
 	}
 
-	@Ignore
+	@Disabled
 	// Reason ignored when running CI it generates this error:
 	//    java.awt.HeadlessException:
 	// 	  No X11 DISPLAY variable was set, but this program performed an operation which requires it.
 	@Test
-	public void testSourceFilesConstructorDirectoryNumberOfFiles() throws Exception {
+	void testSourceFilesConstructorDirectoryNumberOfFiles() throws Exception {
```

Comment:
Shoudl we move this in the GUI project?

Reply:
I removed the @Disabled and it now runs cleanly.